code/obj/machinery/law_racks.dm

```diff
@@ -501,7 +502,13 @@
 			if(!module)
 				if (last_laws[i])
 					//load the law number and text from our saved law list
-					removed_laws += "<del class=\"alert\">[last_laws[i]["number"]]: [last_laws[i]["law"]]</del>"
+					var/lawtext = last_laws[i]["law"]
+					if (islist(lawtext))
+						for (var/law in lawtext)
```

Comment:
shouldn't `var/removed_law_offset = 0` be here instead? otherwise removing two ion'd modules at once would give the wrong offset. Note you can remove multiple modules at once by badly damaging the rack.

Reply:
It turned out to not quite be that simple, but it now does this
![image](https://user-images.githubusercontent.com/20713227/168497531-38bb7370-a2d0-4dac-a882-3a9557cc5290.png)
so it seems good